src/uct/sm/mm/base/mm_iface.h

```diff
@@ -235,7 +235,7 @@ typedef struct uct_mm_iface {
  * @param _cfg_prefix   Prefix for configuration variables.
  */
 #define UCT_MM_TL_DEFINE(_name, _md_ops, _rkey_unpack, _rkey_release, \
-                         _cfg_prefix) \
+                         _cfg_prefix, _cfg_table) \
```

Comment:
pls, add description of `_cfg_table` parameter in doc

Reply:
added